contents/handbook/engineering/deployment-strategy.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Deployment Strategy
+sidebar: Handbook
+showTitle: true
+---
+
+The way we deploy is a key _product_ decision.
+
+* 'Cloud' = PostHog Cloud, we host it for you in *PostHog's* AWS
+* 'Private Cloud' = You host it, we manage it in your AWS / GCP without needing access to your data
+* 'On Premise' = You host it on your own physical servers / somewhere that PostHog can't maintain it for you remotely
+
+We prioritize like this, where 1 = top priority.
+
+<span class="table-borders">
+
+| | Individuals (Free) | Team (small _or_ big company) | Enterprise (department-wide) |
+|---|---|---|---|
+|Cloud|1|5|6|
+|Private Cloud||2|3|
+|On Premise|4|||
+
+</span>
+
+## Why prioritize this stuff at all?
```

Comment:
Who is this page for? It's under the handbook and has instructions for us internally but then uses terms like "we host it for you"